desktop/src/app.rs

```diff
@@ -91,6 +94,20 @@ impl App {
 				};
 				self.send_or_queue_web_message(bytes);
 			}
+			DesktopFrontendMessage::Init => {
```

Comment:
We need to ensure that all auto saved documents are loaded at this point. You assume this message is handled after Portfolio::Init, this works because of the current ordering of message dispatch in desktop wrapper, but this assumption should not be made in my opinion. Maybe a special `FrontendMessage::OpenDocumetnsSuppliedAtStartup` (I'm tired, could not think of a better name) would be a better solution. Could also be used for opening documents from the URL in web.